frontend/src/views/web/permission-denied.html

```diff
@@ -3,15 +3,26 @@
         <div class="col s12 m6 offset-m3">
             <div class="card horizontal ev-md-container ev-card-panel ev-z-depth-5 ">
                 <div class="card-image">
-                    <img src="dist/images/email.png" height="100">
+                    <img src="dist/images/email.png" height="125">
                 </div>
                 <div class="card-stacked">
                     <div class="card-content">
-                        <p><strong>Unable to create or select a team.</strong>
-                            <br> {{perm.emailError}}</p>
+                        <p><strong>Unable to create or select a team.</strong><br>
+                        Hi {{perm.user.username}}!
+                        <br> A <strong>verification</strong> link has been sent to {{perm.user.email}}.
+                        <br><strong>Note:</strong> In order to proceed ahead, verify your account by clicking on the link in the email.
```

Comment:
Remove br form here and add some margin to this section (Add space between note and above content) @Obtin

Reply:
![1](https://user-images.githubusercontent.com/25207205/34643780-71e79ef8-f350-11e7-8102-00feb4966f08.png)